v2/fleet-local/control/mesos-master@.service

```diff
@@ -26,7 +26,7 @@ ExecStart=/usr/bin/sh -c "sudo /usr/bin/docker run \
   --log_dir=/var/log/mesos \
   --quorum=3 \
   --work_dir=/var/lib/mesos/master \
-  --zk=zk://${ZOOKEEPER_ENDPOINT}/mesos"
+  --zk=zk://`etcdctl get /environment/ZOOKEEPER_ENDPOINT`/mesos"
```

Comment:
You might want to use the $() syntax instead of backticks to keep things consistent @chr0n1x unless that syntax doesn't work for running etcdctl.